Neos.Flow/Classes/Persistence/Doctrine/PersistenceManager.php

```diff
@@ -84,19 +84,45 @@ public function injectLogger(LoggerInterface $logger): void
      * Commits new objects and changes to objects in the current persistence
      * session into the backend
      *
-     * @param boolean $onlyAllowedObjects If true an exception will be thrown if there are scheduled updates/deletes or insertions for objects that are not "allowed" (see AbstractPersistenceManager::allowObject())
+     * @param boolean $onlyAllowedObjects If true an exception will be thrown if there are scheduled updates/deletes or insertions for objects that are not "allowed" (see AbstractPersistenceManager::allowObject()). Deprecated: Use `persistOnlyAllowedObjects()` instead.
      * @return void
      * @throws PersistenceException
      * @api
      */
     public function persistAll(bool $onlyAllowedObjects = false): void
+    {
+        if ($onlyAllowedObjects === true) {
+            $this->persistOnlyAllowedObjects();
+            return;
+        }
+        if (!$this->entityManager->isOpen()) {
```

Comment:
Should this "isOpen" check go into a separate method, to avoud duplicated code

Reply:
Not sure it makes sense here, since this is not duplicated anywhere and the log message is specific to the `persistAll()` call